docs/gangplank/usage.md

```diff
@@ -225,40 +250,39 @@ The JobSpec defines discrete, units of work as a "stage". Each stage supports fe
 
 To illustrate this, consider:
 ```yaml
-- id: base Stage
-  build_artifacts:
-  - base
-  post_commands:
-  - cosa kola run --basic-qemu-scenarios
-  request_artifacts:
-  - oscontainer
-  request_cache_repo: true
+
+- id: ExecOrder 1 Stage
+  description: Stage 1 execution base
+  build_artifacts: [base]
+  execution_order: 1
   request_cache: true
-  return_cache: true
-  return_cache_repo: true
-- id: oscontainer
-  build_artifacts:
-  - ostree
-  require_cache: true
-  require_cache_repo: true
-- id: clouds
-  concurrent: true
-  build_artifacts:
-  - aws
-  - gcp
-  required_artifacts:
-  - qemu
-- id: finalize
-  build_artifacts:
-  - finalize
+  request_cache_repo: true
+
+- id: ExecOrder 5 Stage
+  description: Stage 5 execution aws
+  require_artifacts: [qemu]
+  build_artifacts: [aws]
+  execution_order: 5
+
+- id: ExecOrder 5 Stage
```

Comment:
Is the `id` supposed to be unique per stage?

Reply:
It doesn't have to be, no.